engine/dlib/src/dlib/double_linked_list.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2025 The Defold Foundation
```

Comment:
The resource chunk cache has some unit tests, but now that the api is formalized, we need to add tests for this container too.

Reply:
Added test_double_linked_list.cpp